diff --git a/src/main/java/org/example/Main.java b/src/main/java/org/example/Main.java
index 3f4c5ac..2388f48 100644
--- a/src/main/java/org/example/Main.java
+++ b/src/main/java/org/example/Main.java
@@ -36,10 +36,10 @@ import java.util.List;
 
 public class Main {
     //openssl req -x509 -newkey rsa:4096 -keyout prk.ss.pem -out cert.ss.pem -sha256 -days 365 -nodes
-    private static final String PATH_TO_PRIVATE_KEY = "src/main/resources/prk.ss.pem";
-    private static final String PATH_TO_PUBLIC_KEY = "src/main/resources/cert.ss.pem";
-    private static String PATH_TO_SELF_DESCRIPTION = "src/main/resources/vc.json";
-    private static String PATH_TO_SIGNED_SELF_DESCRIPTION = "src/main/resources/sd.signed.json";
+    private static final String PATH_TO_PRIVATE_KEY = "prk.ss.pem";
+    private static final String PATH_TO_PUBLIC_KEY = "cert.ss.pem";
+    private static String PATH_TO_SELF_DESCRIPTION = "vc.json";
+    private static String PATH_TO_SIGNED_SELF_DESCRIPTION = "sd.signed.json";
 
     static String readFile (String path) throws IOException {
         return Files.readString(Path.of(path));
